project/task1/handleWindowPoints.py

```diff
@@ -0,0 +1,12 @@
+def handleWindowPoints(windowPoints): 
+    xList = []
+    yList = []
+    for p in windowPoints:
+        xList.append(p.getX())
+        yList.append(p.getY())
+    maxX = max(xList)
+    minX = min(xList)
+    maxY = max(yList)
+    minY = min(yList)
+    D = maxX - minX + maxY - minY
+    return (D, xList, yList)
```

Comment:
Just let it there for easier mutual understanding. At least they're not really bad.
Use in case of get fixation centroid point.
I'll improve code once the task correctness is ok.